docs-new/docs/graphql/core/actions/codegen/python-flask.mdx

```diff
@@ -21,8 +25,8 @@ to build a Python & Flask API for JWT authentication.
 
 We assume a `user` table with fields `email` and `password`.
 
-We create two `actions <create_actions>` and
-`custom types <custom_types>`:
+We create two [actions](/graphql/core/actions/create.mdx) and
+[custom types](/graphql/core/actions/types/index.mdx):
```

Comment:
Broken file path - should be `types.mdx` - due to removed of directory as only index file exists